test/test.jl

```diff
@@ -266,6 +266,7 @@ function unittest_testsuite(Backend, backend_str, backend_mod, BackendArrayT; sk
     function f(KernelAbstractions.@context, a)
         I = @index(Global, Linear)
         a[I] = 1
+        return # ??
```

Comment:
Is this valid? This function isn't defined in a macro, but used just below in a `@kernel` function...

Reply:
Yeah that'd valid